Assets/MixedRealityToolkit/_Core/Devices/OpenVR/GenericOpenVRController.cs

```diff
@@ -19,36 +20,103 @@ public GenericOpenVRController(TrackingState trackingState, Handedness controlle
-        //private MixedRealityPose currentGripData = MixedRealityPose.ZeroIdentity;
-
-        #region Base override configuration
-
-        /// <summary>
-        /// Mapping method to expose the Unity Input Manager mapping configuration
-        /// </summary>
-        public override InputManagerAxis[] ControllerAxisMappings => ControllerInputAxisMappingLibrary.GetInputManagerAxes(GetType().FullName);
-
-        /// <summary>
-        /// Mapping method to expose this controllers Unity Input Manager mapping array
-        /// </summary>
-        public virtual string[] VRInputMappings => ControllerInputAxisMappingLibrary.GetInputManagerMappings(GetType().FullName);
-
-        #endregion Base override configuration
+        public static readonly MixedRealityInteractionMapping[] DefaultLeftHandedInteractions =
+        {
+            // Controller Pose
+            new MixedRealityInteractionMapping(0, "Spatial Pointer", AxisType.SixDof, DeviceInputType.SpatialPointer, MixedRealityInputAction.None),
+            // HTC Vive Controller - Left Controller Trigger (7)
+            // Oculus Touch Controller - Axis1D.PrimaryIndexTrigger
+            // Valve Knuckles Controller - Left Controller Trigger
+            new MixedRealityInteractionMapping(1, "Trigger Position", AxisType.SingleAxis, DeviceInputType.Trigger, MixedRealityInputAction.None, KeyCode.None, ControllerMappingLibrary.MIXEDREALITY_AXIS9),
+            // HTC Vive Controller - Left Controller Trigger (7)
+            // Oculus Touch Controller - Axis1D.PrimaryIndexTrigger
+            // Valve Knuckles Controller - Left Controller Trigger
+            new MixedRealityInteractionMapping(2, "Trigger Touch", AxisType.Digital, DeviceInputType.TriggerTouch, MixedRealityInputAction.None, KeyCode.JoystickButton15),
+            // HTC Vive Controller - Left Controller Trigger (7)
+            // Oculus Touch Controller - Axis1D.PrimaryIndexTrigger
+            // Valve Knuckles Controller - Left Controller Trigger
+            new MixedRealityInteractionMapping(4, "Trigger Press", AxisType.Digital, DeviceInputType.TriggerPress, MixedRealityInputAction.None, KeyCode.None, ControllerMappingLibrary.MIXEDREALITY_AXIS9),
+            // HTC Vive Controller - Left Controller Grip Button (8)
+            // Oculus Touch Controller - Axis1D.PrimaryHandTrigger
+            // Valve Knuckles Controller - Left Controller Grip Average
+            new MixedRealityInteractionMapping(5, "Grip", AxisType.SingleAxis, DeviceInputType.Trigger, MixedRealityInputAction.None, KeyCode.None, ControllerMappingLibrary.MIXEDREALITY_AXIS11),
+            // HTC Vive Controller - Left Controller Trackpad (2)
+            // Oculus Touch Controller - Axis2D.PrimaryThumbstick
+            // Valve Knuckles Controller - Left Controller Trackpad
+            new MixedRealityInteractionMapping(10, "Trackpad-Thumbstick Position", AxisType.DualAxis, DeviceInputType.Touchpad, MixedRealityInputAction.None),
```

Comment:
Still my question from before: what does this ID mean, and is there a reason they aren't sequential? It doesn't match the 10 on the link you provided (https://docs.unity3d.com/Manual/OpenVRControllers.html), nor do any of the others.

10 here is `Trackpad-Thumbstick Position`; 10 there is Right Controller Trigger.

It's also not used at the moment:
![image](https://user-images.githubusercontent.com/3580640/43109235-9714a3a2-8e9a-11e8-882f-d0e0dbd3ba6d.png)

Reply:
I'll double check the id's for all the controllers, they are ours and should be sequential